### PR TITLE
Fix: Convert template to JSX (fixes #71)

### LINKED ISSUE #71: Convert to JSX
Convert assessmentResults.hbs to JSX.

### PR DESCRIPTION
**Pull request by swashbuck**
Fixes #71 

### Fix
* Converts `assessmentResults.hbs` to `assessmentResults.jsx`
* Adds a new default value for the retry button text in `properties.schema`: _globals._components._assessmentResults.retryText

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: templates/assessmentResults.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <div className="component__widget assessmentresults__widget">
+
+        {_isRetryEnabled &&
+          <div className="component__feedback assessmentresults__feedback">
+            <div className="component__feedback-inner assessmentresults__feedback-inner">
+
+              {retryFeedback &&
+                <div className="assessmentresults__retry-feedback">
+                  <div className="assessmentresults__retry-feedback-inner">
+                    {compile(retryFeedback, props)}
```
Comment (chris-steele) [inline templates/assessmentResults.jsx:26]: Looks good @swashbuck. My only comment is that I think the preference is to use `dangerouslySetInnerHTML={{ __html: compile(retryFeedback, props)}}`, but it's probably fine either way.

Comment (swashbuck) [reply templates/assessmentResults.jsx]: Good callout.  I've made this change.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by joe-allen-89 — APPROVED** [review]
Tested with FW v5.22.8, working as expected.

**Review by chris-steele — APPROVED** [review]

**Pull request merged**